denizen_scripts/behrcraft/repo-link/Commands/Essentials/Essentials Heal.dsc

```diff
@@ -7,7 +7,7 @@ Heal_Command:
     permission: Behr.Essentials.Heal
     tab complete:
         - if <player.groups.contains[Moderation]>:
-            - define Blacklist <list[<player>]>
+            - define Blacklist <[<player>]>
```

Comment:
i see what you meant here but this isn't a definition of `<player>`, just `<player>` here
```suggestion
            - define Blacklist <player>
```